frontend/packages/console-shared/src/components/status/status.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import * as History from 'history';
+import { Link } from 'react-router-dom';
+import { Popover } from '@patternfly/react-core';
+import { Icon, Button } from 'patternfly-react';
+
+type StatusProps = {
+  icon?: string;
+  children: React.ReactNode;
+};
+
+export const Status = ({ icon, children }: StatusProps) => (
+  <>
+    {icon && <Icon type="pf" name={icon} className="kubevirt-status__icon" />} {children}
```

Comment:
Should the space use `{' '}` to ensure it is rendered? But then you need to do more logic to only include the space of the icon is rendered.

Not exactly sure if the purpose but is it supposed to allow any icon as status?

Reply:
Originally the space was not included and margin styling was used to ensure the space is present. I'd prefer using space as it adjusts to the font size. I'll fix this to include the space only when icon is present.